modules/custommutator/bech32.cpp

```diff
@@ -284,7 +284,7 @@ inline unsigned char LowerCase(unsigned char c)
 }
 
 /** Return indices of invalid characters in a Bech32 string. */
-bool CheckCharacters(const std::string& str, std::vector<int>& errors)
+bool CheckCharacters(const std::string_view& str, std::vector<int>& errors)
```

Comment:
`string_view`s are lightweight and are best passed by value (here and elsewhere).

```suggestion
bool CheckCharacters(std::string_view str, std::vector<int>& errors)
```

Reply:
Fixed!